frontend/src/employee-frontend/components/person-shared/PersonDetails.tsx

```diff
@@ -185,6 +192,20 @@ export default React.memo(function PersonDetails({
     })
   }
 
+  const onDuplicate = () => {
+    setDuplicateOngoing(true)
+    void duplicatePerson(person.id).then((response) => {
+      setDuplicateOngoing(false)
+      if (response.isSuccess) {
+        const redirectUrlBase =
+          getAge(person.dateOfBirth) >= CHILD_AGE
```

Comment:
Would it be better to not allow duplicating adults in the first place?

Reply:
Fixed